models/public/hrnet-w32-human-pose-estimation/model.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from config import cfg
+import models
+import torch
+
+
+class HpeHRNet(torch.nn.Module):
+    def __init__(self, cfg, weights):
+        super().__init__()
+        self.impl = models.pose_higher_hrnet.PoseHigherResolutionNet(cfg)
+        checkpoint = torch.load(weights, map_location='cpu')
+        self.impl.load_state_dict(checkpoint)
+        self.impl.eval()
+        self.pool = torch.nn.MaxPool2d(kernel_size=5, stride=1, padding=2)
+
+    def forward(self, image):
+        outputs = self.impl(image)
```

Comment:
could you please add some comments about what and why do you do in this class (for clearance in users' minds :))